primitiv/_function.pyx

```diff
@@ -17,12 +17,13 @@ class functions:
 
     @staticmethod
     def raw_input(shape, vector[float] data, Device device = None, Graph g = None):
-        if device is None:
-            device = Device.get_default()
+        cdef CppDevice *dev_p = NULL
```

Comment:
Maybe it is better to implement a utility function to obtain correct values:
```python
cdef get_cpp_device(Device dev):
  return dev.wrapped if dev is not None else nullptr

cdef get_cpp_graph(Graph g):
  return g.wrapped if g is not None else nullptr
```
Then we can implement this function using just one statement:
```python
return wrapNode(
    func_input_node(
      normShape(shape).wrapped, data, get_cpp_device(dev), get_cpp_graph(g)))
```